src/t8_cmesh/t8_cmesh_types.h

```diff
@@ -59,8 +59,9 @@ typedef struct t8_cprofile t8_cprofile_t; /* Defined below */
   +T8_ECLASS_MAX_EDGES /* Used to store which face is linked to which surface */
 #define T8_CMESH_OCC_FACE_PARAMETERS_ATTRIBUTE_KEY \
   T8_CMESH_OCC_FACE_ATTRIBUTE_KEY + 1 /* Used to store face parameters */
+#define T8_CMESH_LAGRANGE_POLY_DEGREE T8_CMESH_OCC_FACE_PARAMETERS_ATTRIBUTE_KEY + T8_ECLASS_MAX_FACES + 1
```

Comment:
```suggestion
#define T8_CMESH_LAGRANGE_POLY_DEGREE T8_CMESH_OCC_FACE_PARAMETERS_ATTRIBUTE_KEY + T8_ECLASS_MAX_FACES
```
The +1 is not needed here